cmd/root.go

```diff
@@ -122,17 +124,25 @@ the latest stable version will be downloaded from the repository.`,
 			originHelpFunc(cmd, args)
 			return
 		}
+		env, err := environment.InitEnv(repoOpts)
 		cmd, n, e := cmd.Root().Find(args)
-		if (cmd == rootCmd || e != nil) && len(n) > 0 {
+		if (cmd == rootCmd || e != nil) && len(n) > 0 && err != nil {
 			externalHelp(env, n[0], n[1:]...)
 		} else {
 			cmd.InitDefaultHelpFlag() // make possible 'help' flag to be shown
 			_ = cmd.Help()
 		}
+		if err != nil {
+			fmt.Printf("\ninit local env meet error: %s\n", err)
```

Comment:
BTW, I want to delete this line. Users can get this error when they try to execute some commands.

Reply:
I think it's ok to remove it.